bittensor/extrinsics/unstaking.py

```diff
@@ -121,14 +121,12 @@ async def unstake_extrinsic(
     with bittensor.__console__.status(
         f":satellite: Syncing with chain: [white]{subtensor.network}[/white] ..."
     ):
-        old_balance, old_stake = await asyncio.gather(
-            subtensor.get_balance(wallet.coldkeypub.ss58_address),
-            subtensor.get_stake_for_coldkey_and_hotkey(
-                coldkey_ss58=wallet.coldkeypub.ss58_address, hotkey_ss58=hotkey_ss58
-            ),
+        old_balance = await subtensor.get_balance(wallet.coldkeypub.ss58_address)
+        old_stake = await subtensor.get_stake_for_coldkey_and_hotkey(
+            coldkey_ss58=wallet.coldkeypub.ss58_address, hotkey_ss58=hotkey_ss58
```

Comment:
Did using asyncio.gather break this?

Reply:
Yes it was. I'm not sure if this will happen again after optimization. As discussed offline, let's leave this to the optimization stage.